lib/rules/declaration-property-value-no-unknown/__tests__/index.js

```diff
@@ -49,6 +49,9 @@ testRule({
 		{
 			code: 'a { font-size: max(1rem, 3rem); }',
 		},
+		{
+			code: '/* stylelint-disable */ a { filter: alpha(opacity=30); }',
+		},
```

Comment:
[suggest] This case tests `/* stylelint-disable */` rather than this rule, so only the reject case seems enough to me. What about removing the case to avoid making developers confused?

```suggestion
```

Reply:
Thank you for pointing this out, I was also unsure about this one :)